app/models/Product.scala

```diff
@@ -0,0 +1,3 @@
+package models
+
+case class Product(service: String, environment: String, serverTime: String)
```

Comment:
Research companion objects in scala and add the implicit  serializer to this file.

Reply:
I made sure to research it and implement it!